addons/website_event_track/views/event_track_templates_agenda.xml

```diff
@@ -4,7 +4,20 @@
 <!-- Revamped agenda : Will need to replace agenda_online from website_event_track in master -->
 <template id="agenda_online" name="Track Online: Agenda">
     <t t-call="website_event.layout">
-        <div class="o_wevent_online o_weagenda_index">
+        <!-- No tracks -->
+        <div class="container" t-if="not tracks_by_days">
```

Comment:
Just for styling, I would put the t-if attribute first (ease to understand code flow when reading template), like
```
<div t-if="blah" class="..">
    ...
</div>
<div t-else="" class="...">
    ...
</div>
```